src/main/proto/com/google/api/codegen/samplegen/v1/sample_config_v1.proto

```diff
@@ -71,19 +74,24 @@ message SampleSpec {
   // generators should also generate the "default" calling-pattern
   // sample for this RPC if it was not included by any of the
   // previous matches.
-  repeated string calling_patterns = 6;
+  //
+  // Note: the language prefix and `default` calling patterns are not
```

Comment:
Change this note to be a TODO to support them

Reply:
It wouldn't be hard to support `default` I think. But why is it necessary? Wouldn't leaving out this field be an appropriate way to tell the generator to use the `default` one?

I mean, if people do `int i;`, `i == 0` is a proper default value. Do people want to do `int i = int.default` at all?

Reply:
Yes, leaving out the field is an appropriate way. But we also need to support people wanting the default _as well as_ something else. eg `[default, longrunning_start_then_cancel]`